go/oasis-node/cmd/common/flags/flags.go

```diff
@@ -98,7 +110,9 @@ func init() {
 	DebugTestEntityFlags.Bool(CfgDebugTestEntity, false, "use the test entity (UNSAFE)")
 	_ = DebugTestEntityFlags.MarkHidden(CfgDebugTestEntity)
 
-	EntityFlags.StringP(cfgEntity, "e", "", "Path to directory containing entity private key and descriptor")
+	EntityFlags.StringP(cfgEntity, "e", "file", "Backing for of the entity. Options: [file, ledger]")
```

Comment:
Hm this seems like a confusing choice for the signer backend? Shouldn't it be something like `cfgSigner` and be named `signer`? If not needed, the old `entity` option should be removed instead of being repurposed.

Reply:
Also `EntityFlags` should now probably be `SignerFlags`?